docs/content/deployment/guides/aws.mdx

```diff
@@ -78,6 +78,23 @@ def my_job():
 
 [Fargate tasks only support certain combinations of CPU and Memory.](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html)
 
+### Customizing the Launched Run's Task
+
+By default, the <PyObject module="dagster_aws.ecs" object="EcsRunLauncher" /> uses the properties of the current ECS task to determine the configuration for each launched run's task. For example, the launched run will use the same ECS cluster, subnets, security groups, and launch type (e.g. Fargate or EC2) for each new task that it creates.
+
+To adjust the configuration of the launched run's task, set the `run_task_kwargs` config field to a dictionary with additional key-value pairs that should be passed into the `run_task` boto3 API call. For example, to launch new runs in EC2 from a task running in Fargate, you could apply this configuration:
+
+```yaml
+run_launcher:
+  module: "dagster_aws.ecs"
+  class: "EcsRunLauncher"
+  config:
+    run_task_kwargs:
+      launchType: "EC2"
+```
+
+See the [boto3 docs](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ecs.html#ECS.Client.run_task) for the full set of available arguments to `run_task`. Note that the keys are in camelCase format since they correspond to arguments in the boto API. Any argument other than `taskDefinition` (which can be overridden separately using the `task_definition` config field) or `overrides` can be used in `run_task_kwargs`.
```

Comment:
There's a lot of important info here that I feel gets lost being in paragraph form. Consider putting this into a list - ex:

```suggestion
Refer to the [boto3 docs](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ecs.html#ECS.Client.run_task) for the full set of available arguments to `run_task`. Additionally, note that:

- Keys are in `camelCase` as they correspond to arguments in boto's API
- All arguments with the exception of `taskDefinition` and `overrides` can be used in `run_task_kwargs`
```

Reply:
As the word `config` is in here quite a bit, I found this confusing because I wasn't sure where I was supposed to be looking:

> `taskDefinition` (which can be overridden separately using the `task_definition` config field)

It took me a second to figure out it was referring to `run_launcher.config.task_definition`. Is using dot notation to be more specific an okay practice here?

Reply:
Agreed completely! I'll add this as a 'todo' item in the (for now, internal) style guide I have going.